CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Contribute
+
+Want to show Sinatra some love? Help out by contributing!
+
+## Find a bug?
+
+Log it in our [issue tracker][ghi] or send a
+note to the [mailing list][ml].
+Be sure to include all relevant information, like the versions of Sinatra and
+Ruby you're using. A [gist](http://gist.github.com/) of the code that caused
+the issue as well as any error messages are also very helpful.
+
+## Need help?
+
+The [Sinatra mailing list][https://groups.google.com/forum/#!forum/sinatrarb] has
```

Comment:
Should  the link here be in parentheses `()`? GitHub's rich-diff renderer doesn't seem to parse this properly.